linker/shared/src/main/scala/org/scalajs/linker/backend/javascript/Printers.scala

```diff
@@ -503,22 +513,23 @@ object Printers {
           if (isStat)
             print('(') // force expression position for the object literal
           print('{')
-          indent()
           println()
+          indent()
```

Comment:
This swap does not seem needed, and makes this spot inconsistent with other places that increase the indentation (for example `printBlock`). Consider reverting.

Reply:
The swap is indeed not needed (`indent()` and `println()` are now independent of each other). I originally did the swap since it seemed more logical (what a human would write, newline, then indent). However, it is indeed less consistent with the rest of the class (where the swap cannot be so easily done due to pre element println). So I think sticking with the existing order is fine.